reg/drone/service/sensor/battery/Parameters.0.1.uavcan

```diff
@@ -0,0 +1,54 @@
+# Smart battery parameter message. It is mostly intended for automated battery charging and maintenance systems.
+# This message is modeled after the Smart Battery Data Specification (SBS) and the MAVLink battery status messages.
+#
+# The values carried by this message are either constant or slow-changing, so, generally, the publishing frequency
+# should not be higher than 0.2 Hz, and the priority should be either OPTIONAL or SLOW.
+#
+# All parameters are required unless specifically stated otherwise.
+
+truncated uint64 unique_id
+# A statistically unique number that can be used to differentiate this exact battery from others.
+# Needed for logging purposes. If the battery supports SBS, this value may be computed as CRC64WE of
+# (ManufacturerName + DeviceName + ManufactureDate + SerialNumber).
+# This value shall be invariant to the identity of the reporting node unless it is an integral part of the battery.
+# Vendors may use a constant random number for the 40 most significant bits for vendor identification purposes.
+
+uavcan.si.unit.mass.Scalar.1.0 mass
+# The total mass of the battery, including the packaging, electronics, cabling, and all auxiliary items, if any.
+# May be used for predicting the kinematic parameters of the vehicle.
+# NaN if unknown.
+
+uavcan.si.unit.electric_charge.Scalar.1.0 design_capacity
+# The maximum total charge of the pack, at 100% SoH, specified by the manufacturer.
```

Comment:
I think we should specify the units {Ah}, or {Wh}

Reply:
> {Ah}, or {Wh}

This is an electric charge, not energy, and we only use SI units (it is encoded in the type name: `uavcan.si.unit.electric_charge...`), so the only option is the coulomb, there is no ambiguity. If you look at the definition of [`uavcan.si.unit.electric_charge.Scalar.1.0`](https://github.com/UAVCAN/public_regulated_data_types/blob/master/uavcan/si/unit/electric_charge/Scalar.1.0.uavcan) it also says `coulomb`.